resources/guest_configs/microvm-kernel-ci-aarch64-4.14.config

```diff
@@ -2482,7 +2482,7 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
 CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
 CONFIG_HAVE_C_RECORDMCOUNT=y
 CONFIG_TRACING_SUPPORT=y
-# CONFIG_FTRACE is not set
+CONFIG_FTRACE=y
```

Comment:
We want to keep our downstream modifications to the configs as patches: https://github.com/firecracker-microvm/firecracker/tree/main/resources/guest_configs/patches .

Reply:
Thx, done